src/main.ts

```diff
@@ -128,6 +128,19 @@ export class Minifier {
         if (isExternal || lhsIsModule) return output + this._ident(pae.name);
         return output + this._renameIdent(pae.name);
       }
+      case ts.SyntaxKind.Parameter: {
+        var paramDecl = <ts.ParameterDeclaration>node;
+
+        // if there are modifiers, then we know this is a declaration and an initialization at once
```

Comment:
note that if that same parameter is used in the constructor body, we would have to rename it there too. Renaming function local variables is something we have been relying on uglify to do, but here we have to handle it ourselves.

This will be trickier (needs keeping a locally scoped renaming map) so just add a TODO for now.